projects/packages/my-jetpack/_inc/components/add-license-screen/index.jsx

```diff
@@ -54,7 +58,7 @@ export default function AddLicenseScreen() {
 					<ActivationScreen
 						currentRecommendationsStep={ null }
 						availableLicenses={ availableLicenses }
-						fetchingAvailableLicenses={ fetchingAvailableLicenses }
+						fetchingAvailableLicenses={ isLoading }
```

Comment:
We could rename `isLoading` to `fetchingAvailableLicenses` for simplicity of changes. Not a blocker though

Reply:
I changed it on the request of consistency from @IanRamosC (https://github.com/Automattic/jetpack/pull/36029#discussion_r1507717712) - what do you think?